test/integration/challenges.test.js

```diff
@@ -0,0 +1,120 @@
+const chai = require('chai')
+const { expect } = chai
+const chaiHttp = require('chai-http')
+
+const app = require('../../server')
+const authService = require('../../services/authService')
+const addUser = require('../utils/addUser')
+const cleanDb = require('../utils/cleanDb')
+
+const config = require('config')
+const cookieName = config.get('userToken.cookieName')
+
+chai.use(chaiHttp)
+
+describe('Challenges', function () {
+  let jwt
+
+  beforeEach(async function () {
+    const userId = await addUser()
+    jwt = authService.generateAuthToken({ userId })
+  })
+
+  afterEach(async function () {
+    await cleanDb()
+  })
+
+  describe('GET /challenges', function () {
+    it('Should return the available challenges', function (done) {
+      chai
+        .request(app)
+        .get('/challenges')
+        .set('cookie', `${cookieName}=${jwt}`)
+        .end((err, res) => {
+          if (err) {
+            return done(err)
+          }
+          expect(res).to.have.status(200)
+          expect(res.body).to.be.a('object')
+          expect(res.body.message).to.be.equal('Challenges returned successfully!')
+          expect(res.body.challenges).to.be.a('array')
```

Comment:
Also check if the array length is greater than 1 or not since we are checking for length in controller.